packages/timeline-state-resolver/src/integrations/tricaster/timelineStateConverter.ts

```diff
@@ -0,0 +1,155 @@
+import { TimelineState } from 'superfly-timeline'
+import {
+	Mappings,
+	MappingTriCaster,
+	MappingTriCasterType,
+	TSRTimelineObjBase,
+	isTimelineObjTriCasterAudioChannel,
+	isTimelineObjTriCasterDSK,
+	isTimelineObjTriCasterME,
+	isTimelineObjTriCasterMixOutput,
+	MappingTriCasterMixEffect,
+	MappingTriCasterDownStreamKeyer,
+	MappingTriCasterAudioChannel,
+	MappingTriCasterMixOutput,
+	TriCasterAudioChannelName,
+	TriCasterMixEffectName,
+	TriCasterMixOutputName,
+	MappingTriCasterInput,
+	isTimelineObjTriCasterInput,
+	TriCasterInputName,
+} from 'timeline-state-resolver-types'
+import * as _ from 'underscore'
+import { TriCasterState } from './triCasterStateDiffer'
+
+type DeepPartial<T> = { [P in keyof T]?: T[P] extends object ? DeepPartial<T[P]> : T[P] }
+
+export class TimelineStateConverter {
```

Comment:
This converts timeline state for a Tricaster, so it would be nice if the name indicated that `TricasterTimelineStateConverter`